test/helm3/ct.yaml

```diff
@@ -0,0 +1,20 @@
+debug: true
+target-branch: master
+chart-dirs:
+  - stable
+  - staging
+excluded-charts:
+  - common
+  - dex-controller # Moved to a different helm repo
```

Comment:
All [of these charts](https://github.com/mesosphere/charts/pull/733/files#diff-a139d5b87837ab46aab5151c40082121R9-R28) failed to install with ct for helm2. Is this no longer the case with ct helm3?

Reply:
I would have to test these 1 by 1. The reasons for the lack of testing an be complicated, and there are jiras on a few of these.